### DISCUSSION TITLE
Recharge estimation without specific yield

### DISCUSSION THREAD

**Discussion by robinkeegan** [pastas/pastas#671] [Q&A] [answered]
My understanding is that transfer function modelling combines different stresses together to produce a signal such as a hydrograph. So when we are estimating recharge from a rainfall stress we are finding the change in hydraulic head due to rainfall. Once we have found the change in head due to rainfall, should this be multiplied by specific yield to calculate recharge (similar to the water table fluctuation method)? I notice that the example and some published papers don't multiply by specific yield when calculating recharge and I am curious how pastas can be used to estimate recharge without specific yield.

**Comment by raoulcollenteur** [comment, marked as answer]
Hi @robinkeegan,

Thanks for your interest in trying Pastas to estimate recharge. For recharge estimation, there are different recharge models available. These all use a (soil) water balance approach to estimate recharge to the groundwater, and finally the contribution of recharge to the head fluctuations, so no need to estimate the specific yield. Here's the open access paper where we describe the method: https://hess.copernicus.org/articles/25/2931/2021/ . Let me know if that helps! Happy to answer your questions.
Cheers,
Raoul

**Reply by robinkeegan** [reply]
Thankyou @raoulcollenteur that is very helpful!
